### PR TITLE
Added an equipment piece with stat boosts for GCI task

### PR DESCRIPTION
**Pull request by arpitkamboj**
Added an equipment piece with stat boosts: "Gold Strength n Luck" - A golden ring that makes you lucky and increases strength.

### CODE REVIEW

File: assets/prefabs/Equipment/Gold/GoldStrengthnLuck.prefab [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"parent" : "engine:iconItem",
```
Comment (Cervator) [inline assets/prefabs/Equipment/Gold/GoldStrengthnLuck.prefab:1]: FYI you somehow lost the leading `{` here. No big deal so I fixed it real quick :-)

**Comment by Cervator** [comment]
Merged! Thanks :-)

File: assets/prefabs/Equipment/Gold/GoldStrengthnLuck.prefab (hunk 1 above)
Comment (arpitkamboj) [reply assets/prefabs/Equipment/Gold/GoldStrengthnLuck.prefab:1]: oops....
anyways thnx :D

**Comment by arpitkamboj** [comment]
Thank you
-Arpit

**Pull request merged**